### PR TITLE
Fix issue #2260

### PR DESCRIPTION
**Pull request by pubudu91**
This PR fixes issue #2260 by checking a variable before using it to verify whether it is initialized.

### CODE REVIEW

File: modules/ballerina-core/src/main/java/org/ballerinalang/bre/BLangExecutor.java [hunk 1]
```diff
@@ -1073,6 +1073,10 @@ private void assignValueToArrayMapAccessExpr(BValue rValue, ArrayMapAccessExpr l
         if (!(accessExpr.getRExpr().getType() == BTypes.typeMap)) {
             BArray arrayVal = (BArray) accessExpr.getRExpr().execute(this);
 
+            if (arrayVal == null) {
+                throw new BallerinaException("field " + accessExpr.getSymbolName() + " is not initialized");
```
Comment (SupunS) [inline modules/ballerina-core/src/main/java/org/ballerinalang/bre/BLangExecutor.java]: Shall we reword this to "variable 'x' is null" as in [1] to maintain the consistency in the error messages?

[1] https://github.com/ballerinalang/ballerina/blob/master/modules/ballerina-core/src/main/java/org/ballerinalang/bre/BLangExecutor.java#L734

Comment (pubudu91) [inline modules/ballerina-core/src/main/java/org/ballerinalang/bre/BLangExecutor.java]: Changed the error message to conform with the format given above. Please check.
